app/code/Magento/Wishlist/view/frontend/web/js/add-to-wishlist.js

```diff
@@ -130,7 +132,6 @@ define([
          * @param {Object} array2
          * @return {Object}
          * @private
-         * @deprecated
```

Comment:
Unfortunately, we may not remove the deprecation flag from the methods that were deprecated in the past. Also, it's recommended to avoid using the deprecated methods. 

I would recommend you to take a look at the following PR in the past, where the deprecation has been introduced and the removing invocation has been deleted.

https://github.com/magento/magento2/commit/b801020c010e60a69d12611bf83fc03eb0676840#diff-9c7ce06b9bcfcfd8196d6f5d96b490baL115

There's a high chance that we will introduce another bug that was fixed in the mentioned PR.

Thank you